common/test/acceptance/tests/helpers.py

```diff
@@ -199,6 +199,20 @@ def enable_css_animations(page):
     """)
 
 
+def coordinates_for_scrolling(page, css_selector):
```

Comment:
This method can be used more generically than for scrolling. Maybe a better name would be get_element_coordinates.